src/tests/utils/setup.ts

```diff
@@ -34,6 +35,19 @@ declare global {
    * @returns accessToken for authorization
    */
   function auth(userId: number) : string;
+
+
+  /* eslint-disable-next-line no-var */
+  var db: {
+    /**
+     * Executes specified sql query in test DB.
+     * Might be used in tests to perform some specific database operations
+     *
+     * @param sql - string containing sql to executein test DB
+     * @returns
+     */
+    query: (sql: string) => Promise<unknown>;
+  };
```

Comment:
maybe make make it as function?

Reply:
you mean get rid of `db` wrapper?

Reply:
yes, implement it as a SQL query function to the database

Reply:
and call it like this? `global.executeSql()`?

Reply:
yeah would be nice

Reply:
you can leave it, but it seems that nothing more can be done with the database other than accessing it via sql